src/server.js

```diff
@@ -58,24 +66,74 @@ router.get('/SubmitJira', function (req, res) {
   }
 
   request(options).then(function (response) {
-    let key = response.key;
-    SendJiraAttach(key, fileData);
+    issueKey = response.key;
+    SendJiraAttach(issueKey, fileData);
     res.send(response)
     res.status(200).json(response);
   })
     .catch(function (err) {
       console.log(err);
     })
+  // capture the encoded form data
+  req.on('data', (data) => {
+    console.log(data.toString());
+  });
+  // send a response when finished reading
+  // the encoded form data
+  req.on('end', () => {
+    res.send('ok');
+  });
 });
 
 
+
 // REGISTER OUR ROUTES -------------------------------
 // all of our routes will be prefixed with /api
 app.use('/api', router);
 app.use('/', express.static(testHarnessPath));
 
-// send attachment
-function SendJiraAttach(key, data){
+// create issue id
+function createJiraIssue() {
```

Comment:
When do you call the function  `createJiraIssue`?

Reply:
I think you renamed it from `SendJiraAttach` to `createJiraIssue` without brining over the parameters.

Reply:
no, sorry that's a duplicate code, the create jira issue is embeded now /receivedata route. I just removed the duplicate function. SendJiraAttachment takes care of attaching the file after an ticket id has been issues. there are 3 steps to the process:
1- receive data from client 
2- create jira issue id
3- attach the report file to the created ticket id